style.css

```diff
@@ -45,6 +45,10 @@ body {
   transition: all 0.2s ease;
 }
 
+.option {
+  display: none;
+}
+
```

Comment:
- [ ] Please be aware that the mobile screen sizes have a range between **min 375px** and **mac 786px** so all project cards need to stretch on width to take up the mobile screen width 👍🏼 

### Your Design
![image](https://user-images.githubusercontent.com/68030297/228998992-89471c4e-db81-406a-8cf8-4514b2dc277d.png)

### Figma Design
![image](https://user-images.githubusercontent.com/68030297/228999172-c9c6b55a-6694-4203-bff9-c2212b59287e.png)


## Suggestions
- By avoiding setting a fixed width to the card you will start seeing some good results, currently, the `container1` class has a fixed width of `327px`, if you remove it the card will take the correct width by default 👍🏼

Reply:
I parse the Figma design to build the mobile version. I use All the properties given. And it should be display in screen 375px